packages/component-static-pages/client/pages/ContactUs/EditorialStaff.js

```diff
@@ -25,10 +25,10 @@ const EditorialStaff = () => (
         telephone="+44 [0]1223 855376"
       />
       <StaffCard
-        jobTitle="Senior Editorial Assistant"
-        name="Susanna Richmond"
-        photoURL="https://iiif.elifesciences.org/journal-cms/person%2F2017-07%2Fsusanna.jpg/0,53,1000,566/502,284/0/default.jpg"
-        telephone="+44 [0]1223 855377"
+        jobTitle="Editorial Assistant"
```

Comment:
I think Milly and Emma are now both Senior: 
> I'm pleased to say that Milly and Emma are being promoted to Senior Editorial Assistants, effective October 1. 

Or is this going to be another request ?